noir-projects/aztec-nr/aztec/src/note/note_getter/test.nr

```diff
@@ -73,14 +69,10 @@ unconstrained fn constrain_get_notes_processes_many_notes() {
         notes_to_constrain[0] = Option::some(build_valid_retrieved_note(contract_address, 13));
         notes_to_constrain[1] = Option::some(build_valid_retrieved_note(contract_address, 19));
 
-        let options = NoteGetterOptions::new();
-        let (returned, _) = constrain_get_notes_internal(
-            context,
-            owner,
-            storage_slot,
-            notes_to_constrain,
-            options,
-        );
+        let mut options = NoteGetterOptions::new();
+        options = options.set_owner(owner);
```

Comment:
```suggestion
        let options = NoteGetterOptions::new().set_owner(owner);
```

No need to be overly verbose, the point of these methods is to allow chaining like this.

Reply:
https://github.com/AztecProtocol/aztec-packages/commit/31e1d26cb1fc4fa2eed815ced2a43934dac6c644